src/i18n/messages-fr.po

```diff
@@ -0,0 +1,875 @@
+
+msgid "OK"
+msgstr "OK"
+
+msgid "Profile"
+msgstr "Profil"
+
+msgid "Your name"
+msgstr "Votre nom"
+
+msgid "Your picture"
+msgstr "Votre avatar"
+
+msgid "Status message"
+msgstr "Message de statut"
+
+msgid "What's on your mind?"
+msgstr "Quel est votre état d'esprit ?"
+
+msgid ""
+"Your picture, name and status message identify yourself in calls, chats "
+"and rooms."
+msgstr ""
+"Votre avatar, nom et état d'esprit servent à vous identifier dans les "
+"appels, chat et salle"
+
+msgid "Your ID"
+msgstr "Votre ID"
+
+msgid "Register"
+msgstr "S'enregistrer"
+
+msgid ""
+"Authenticated by certificate. To log out you have to remove your "
+"certificate from the browser."
+msgstr ""
+"Authentifié par certificat. Vous devrez effacer le certificat de  "
+"de votre navigateur pour vous déconnecter."
+
+msgid "Sign in"
+msgstr "Connnexion"
+
+msgid "Create an account"
+msgstr "Créer un compte"
+
+msgid "Sign out"
+msgstr "Deconnexion"
+
+msgid "Manage account"
+msgstr "Gestion du compte"
+
+msgid "Media"
+msgstr "Webcam / Micro"
+
+msgid "Microphone"
+msgstr "Micro"
+
+msgid "Camera"
+msgstr "Webcam"
+
+msgid "Video quality"
+msgstr "Qualité vidéo"
+
+msgid "Low"
+msgstr "Basse"
+
+msgid "High"
+msgstr "Haute"
+
+msgid "HD"
+msgstr "HD"
+
+msgid "Full HD"
+msgstr "Full HD"
+
+msgid "General"
+msgstr "Général"
+
+msgid "Language"
+msgstr "Langue"
+
+msgid "Language changes become active on reload."
+msgstr "Rechargez la page pour prendre en compte le changement de langue."
+
+msgid "Default room"
+msgstr "Salle par défaut"
+
+msgid "Set alternative room to join at start."
+msgstr "Nom de la salle à rejoindre à la connexion"
+
+msgid "Notifications"
+msgstr "Notifications"
+
+msgid "Desktop notification"
+msgstr "Notifications sur le bureau"
+
+msgid "Enable"
+msgstr "Activer"
+
+msgid "Denied - check your browser settings"
+msgstr "Interdit - vérifier les paramètres de votre navigateur"
+
+msgid "Allowed"
+msgstr "Autorisé"
+
+msgid "Sounds for incoming messages"
+msgstr "Son pour les messages entrant"
+
+msgid "Ring on incoming calls"
+msgstr "Sonnerie pour les appels entrant"
+
+msgid "Sounds for users in current room"
+msgstr "Son pour les participants de la salle courante."
+
+msgid "Advanced settings"
+msgstr "Réglages avancés"
+
+msgid "Play audio on same device as selected microphone"
+msgstr "Ecouter l'audio sur le même materiel que le micro"
+
+msgid "Experimental AEC"
+msgstr "AEC (experimental)"
+
+msgid "Experimental AGC"
+msgstr "AGC (experimental)"
+
+msgid "Experimental noise suppression"
+msgstr "Suppresion de bruit (experimental)"
+
+msgid "Max video frame rate"
+msgstr "Image par seconde max"
+
+msgid "auto"
+msgstr "auto"
+
+msgid "Send stereo audio"
+msgstr "Envoyer de l'audio stéréo"
+
+msgid ""
+"Sending stereo audio disables echo cancellation. Enable only if you have "
+"stereo input."
+msgstr ""
+"Envoyer de l'audio en stéréo désactive la suppression de l'écho."
+"Utilisez le uniquement si vous avez un micro stéréo."
```

Comment:
Missing leading whitespace (to have a blank after the . from the sentence before).